src/net_processing.cpp

```diff
@@ -908,6 +908,10 @@ class PeerManagerImpl final : public PeerManager
     /** Process a new block. Perform any post-processing housekeeping */
     void ProcessBlock(CNode& node, const std::shared_ptr<const CBlock>& block, bool force_processing, bool min_pow_checked);
 
+    /** Process compact block txns  */
+    void ProcessCompactBlockTxns(CNode& pfrom, BlockTransactions& block_transactions, Peer& peer, const CNetMsgMaker& msg_maker)
```

Comment:
```suggestion
    void ProcessBlockTxns(CNode& pfrom, const Peer& peer, const BlockTransactions& block_transactions)
```

Just create a new CNetMsgMaker, no need to pass it.

Reply:
Done, thanks!